code/game/machinery/vending.dm

```diff
@@ -1026,6 +1027,7 @@
 					/obj/item/weapon/tool/knife/tacknife = 5)
 
 	prices = list(
+					/obj/item/weapon/gun/projectile/automatic/slaught_o_matic = 199,
```

Comment:
Why the weird price? Let's make it cost the same amount as it does in Cyberpunk - 90$. Nobody is going to pay 200$ for one magazine worth of gun.

```suggestion
					/obj/item/weapon/gun/projectile/automatic/slaught_o_matic = 90,
```